app/experimenter/nimbus-ui/src/components/PageEditBranches/index.test.tsx

```diff
@@ -3,28 +3,62 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from "react";
-import { screen, waitFor, render } from "@testing-library/react";
+import { screen, waitFor, render, fireEvent } from "@testing-library/react";
 import PageEditBranches from ".";
 import FormBranches from "../FormBranches";
 import { RouterSlugProvider } from "../../lib/test-utils";
 import { mockExperimentQuery, MOCK_CONFIG } from "../../lib/mocks";
 
 describe("PageEditBranches", () => {
+  const origConsole = console.log;
+
+  beforeEach(() => {
+    console.log = jest.fn();
+  });
+
+  afterEach(() => {
+    console.log = origConsole;
+  });
+
   it("renders as expected with experiment data", async () => {
     const { mock } = mockExperimentQuery("demo-slug");
     render(<Subject mocks={[mock]} />);
     await waitFor(() => {
       expect(screen.getByTestId("PageEditBranches")).toBeInTheDocument();
       expect(screen.getByTestId("header-experiment")).toBeInTheDocument();
     });
-    expect(screen.getByTestId("equal-ratio")).toHaveTextContent("false");
     expect(screen.getByTestId("FormBranches")).toBeInTheDocument();
     expect(screen.getByTestId("feature-config")).toBeInTheDocument();
     for (const feature of MOCK_CONFIG!.featureConfig!) {
       const { slug } = feature!;
       expect(screen.getByText(slug)).toBeInTheDocument();
     }
   });
+
+  it("handles onSave from FormBranches", async () => {
+    const { mock, data: experiment } = mockExperimentQuery("demo-slug");
+    render(<Subject mocks={[mock]} />);
+    await waitFor(() => {
+      expect(screen.getByTestId("PageEditBranches")).toBeInTheDocument();
+    });
+    fireEvent.click(await screen.findByTestId("save-button"));
+    expect(console.log).toHaveBeenCalledWith("SAVE", {
+      equalRatio: true,
+      featureConfig: experiment!.featureConfig,
+      referenceBranch: experiment!.referenceBranch,
+      treatmentBranches: experiment!.treatmentBranches,
+    });
+  });
+
+  it("handles onNext from FormBranches", async () => {
+    const { mock } = mockExperimentQuery("demo-slug");
+    render(<Subject mocks={[mock]} />);
+    await waitFor(() => {
+      expect(screen.getByTestId("PageEditBranches")).toBeInTheDocument();
+    });
+    fireEvent.click(await screen.getByTestId("next-button"));
```

Comment:
nit: don't need to await this

Reply:
Ah, yeah, I think in my flailings I added the await here before I added the `waitFor` before it